flink-test-utils-parent/flink-table-filesystem-test-utils/src/test/java/org/apache/flink/table/file/testutils/catalog/TestFileSystemCatalogTest.java

```diff
@@ -390,8 +391,7 @@ public void testAlterCatalogMaterializedTable() throws Exception {
 
         CatalogMaterializedTable actualMaterializedTable = (CatalogMaterializedTable) actualTable;
         // validate refresh status
-        assertThat(actualMaterializedTable.getRefreshStatus())
-                .isEqualTo(CatalogMaterializedTable.RefreshStatus.ACTIVATED);
+        assertThat(actualMaterializedTable.getRefreshStatus()).isEqualTo(RefreshStatus.ACTIVATED);
```

Comment:
nit: if this is an enum shouldn't we use `isSameAs`? 
```suggestion
        assertThat(actualMaterializedTable.getRefreshStatus()).isSameAs(RefreshStatus.ACTIVATED);
```